src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/classes/concat_sdp.cpp

```diff
@@ -71,7 +73,11 @@ void ConcatSDPTest::SetUp() {
     // q,k,v
     inputParams.push_back(std::make_shared<ov::op::v0::Parameter>(inType, inputDynamicShapes[0]));
     inputParams.push_back(std::make_shared<ov::op::v0::Parameter>(inType, inputDynamicShapes[0]));
-    inputParams.push_back(std::make_shared<ov::op::v0::Parameter>(inType, inputDynamicShapes[0]));
+    auto v_ps = inputDynamicShapes[0];
+    if (m_isDiffKVHeadSize) {
```

Comment:
I am wondering why we need to handle it via separate flag and not via existing inputShapes param?

Reply:
Because only head size dimension of kv is not same, using `inputShapes` will make the initializer list bigger and more redundant fields. I think using a flag will make the table a little more readable and should be enough to cover the function test.